include/mbedtls/build_info.h

```diff
@@ -53,6 +53,24 @@
 #define _CRT_SECURE_NO_DEPRECATE 1
 #endif
 
+/** \def MBEDTLS_CHECK_RETURN
```

Comment:
I defined this before including `mbedtls_config.h` with the idea that `mbedtls_config.h` could override the definition (for example, `#undef` and `#define` as empty if the warnings are not desired or not supported on a not-exactly-GCC compiler). But upon reflection I think it would be better to define this after including `mbedtls_config.h`, in the more logical place `platform_util.h`, with a provision that if `MBEDTLS_CHECK_RETURN` is already defined then we don't override it.

Reply:
Now this is declared and commented out in `mbedtls_config.h`, and defined if not already defined in `platform_util.h`.